src/commcare_cloud/fab/operations/supervisor.py

```diff
@@ -244,16 +244,9 @@ def _format_env(current_env, extra=None):
 
     all_hosts = env.ccc_environment.sshable_hostnames_by_group['all']
 
-    ret['supervisor_env_vars'] = {}
-    ret['command_prefix'] = ''
-
-    if env.newrelic_djangoagent:
-        host = current_env.get('host_string')
-        webworkers = env.ccc_environment.groups['webworkers']
-        if host in webworkers:
-            ret['command_prefix'] = '%(virtualenv_root)s/bin/newrelic-admin run-program ' % env
-            ret['supervisor_env_vars']['NEW_RELIC_CONFIG_FILE'] = '%(root)s/newrelic.ini' % env
-            ret['supervisor_env_vars']['NEW_RELIC_ENVIRONMENT'] = current_env.get('deploy_env', '')
+    ret['supervisor_env_vars'] = {
+        "TMPDIR": "/opt/tmp"
```

Comment:
@emord FYI for your other PR of moving things to ansible

Reply:
Thanks. hoping to test that again on staging tomorrow and possibly release it